samples/containers/app-poetry/src/main/java/com/squareup/sample/poetryapp/PoemListView.kt

```diff
@@ -25,29 +10,41 @@ import androidx.recyclerview.widget.RecyclerView
 import com.squareup.sample.container.overviewdetail.OverviewDetailConfig
 import com.squareup.sample.container.overviewdetail.OverviewDetailConfig.Overview
 import com.squareup.sample.container.poetryapp.R
-import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
+import com.squareup.sample.poetry.model.Poem
+import com.squareup.workflow1.ui.AndroidViewRendering
 import com.squareup.workflow1.ui.LayoutRunner
-import com.squareup.workflow1.ui.LayoutRunner.Companion.bind
-import com.squareup.workflow1.ui.ViewFactory
 import com.squareup.workflow1.ui.ViewEnvironment
+import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
 
 @OptIn(WorkflowUiExperimentalApi::class)
-class PoemListLayoutRunner(view: View) : LayoutRunner<PoemListRendering> {
+data class PoemListView(
```

Comment:
> The rendering class for FooWorkflow is named FooView

Instead of `FooRendering`? I think it's simpler to just stick with "Rendering".

> If it has a LayoutRunner implementation, that's private and declared in the same file.

Definitely, I assumed these would always be private implementations.